content/en/docs/aboutjetstack/_index.md

```diff
@@ -1,22 +1,23 @@
 ---
 title: "About Jetstack"
 linkTitle: "About Jetstack"
-weight: 90
+weight: 500
 type: "docs"
 ---
 
-cert-manager was originally created by [Jetstack](https://jetstack.io), and
-a lot of the development is still sponsored by them.
+cert-manager was originally created by [Jetstack](https://jetstack.io), who
+still sponsor a lot of cert-manager development.
 
 Jetstack is a fast-growing Kubernetes professional services company that prides
 itself on helping startups and enterprises alike in their path to modern
 cloud-native infrastructure.
 
```

Comment:
proposed:

`cert-manager was originally created by [Jetstack](https://jetstack.io) and donated to the [CNCF Sandbox](https://www.cncf.io/sandbox-projects/). Jetstack remains the primary maintainers of the project and sponsors most of its development.`

`Jetstack is a company focused on helping businesses to build and operate modern cloud native infrastructure with Kubernetes and have been involved in the ecosystem since the inception of the project. Jetstack provide enterprise support and product enhancements for cert-manager.`

Reply:
this is exactly the kind of thing I was looking for, thanks :+1: I've updated now.